### PR TITLE
ZGW-3429: doc: Update README for quick instructions

### PR DESCRIPTION
**Pull request by rzr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,184 @@
...
+cmd_class:  COMMAND_CLASS_NETWORK_MANAGEMENT_BASIC  v2
+cmd:  DEFAULT_SET_COMPLETE
+(...)
 ```
-On all systems:
-```bash
-$ curl -L http://sourceforge.net/projects/plantuml/files/plantuml.1.2019.7.jar/download --output /opt/plantuml.jar
+
+
+Next step it to interact with an other device,
```
Comment (lmolina) [inline README.md:92]:
```suggestion
Now, you can interact with an end-device. The end-device must be prepared with an application,
e.g., zwave_soc_switch_on_off-brd4202a-eu.hex. Use
[Simplicity Studio](https://www.silabs.com/developers/simplicity-studio) to download and flash one
of the available sample applications.
```

File: README.md [hunk 2]
```diff
@@ -1,116 +1,184 @@
...
+end node should be prepared with application firmware
```
Comment (lmolina) [inline README.md:93]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,116 +1,184 @@
...
+(eg: zwave_soc_switch_on_off-brd4202a-eu.hex),
+and then included to Z-Wave network.
```
Comment (lmolina) [inline README.md:95]:
```suggestion
Once the end-device is prepared, the Z/IP Gateway can include it to the Z-Wave network.
```

File: README.md [hunk 4]
```diff
@@ -1,116 +1,184 @@
...
-3. For detailed description on compiling zipgateway please refer to user guide 
-generated in step 2. 
-Open src/doc/html/index.hml in browser
+## HOW TO BUILD Z/IP/GATEWAY
 
-```bash
-$ xdg-open src/doc/html/index.html
-```
-# Quick compiling Z/IP Gateway debian package
+As said above, only 32 bits OS are currently supported.
```
Comment (lmolina) [inline README.md:133]:
```suggestion
The Z/IP Gateway only suports 32-bit operating systems.
```

File: README.md [hunk 5]
```diff
@@ -1,116 +1,184 @@
...
+
+Reference OS is currently debian-9 (EoL), native building is supported (using cmake),
```
Comment (lmolina) [inline README.md:135]:
```suggestion
The reference OS is currently Debian-9 (EoL), native building is supported (using cmake),
```

File: README.md [hunk 6]
```diff
@@ -1,116 +1,184 @@
...
+and you can rely on helper script to setup system and pass tests to generate a debian package
```
Comment (lmolina) [inline README.md:136]:
```suggestion
and you can rely on helper script to setup system and pass tests to generate a Debian package
```

File: README.md [hunk 7]
```diff
@@ -1,116 +1,184 @@
...
+ready to be installed (see above) (This script can be also run from docker or github)
```
Comment (lmolina) [inline README.md:137]:
```suggestion
ready to be installed (see above) (This script can be also run from Docker or GitHub).
```

File: README.md [hunk 8]
```diff
@@ -1,116 +1,184 @@
...
+
+```sh
+sudo apt install make
+./helper.mk help
+./helper.mk setup/raspbian
+./helper.mk
 ```
 
-### Inside docker
+For the record, dependencies are listed in helper.mk,
```
Comment (lmolina) [inline README.md:151]:
```suggestion
The dependencies are listed in helper.mk,
```

File: README.md [hunk 9]
```diff
@@ -1,116 +1,184 @@
...
-```bash
-dev-machine:~/zw-zgw/$ cd docker/armhf_debian_stretch_cross/
-dev-machine:~/zw-zgw/$ make image
-dev-machine:~/zw-zgw/$ docker run -v ~/zw-zgw/:/zgw -it zwave/zgw_armhf_debian_stretch_cross bash
-```
+### BUILD ON HOST
 
-### Inside docker
+To speed up the build, native build can be deported to host using
+different containerization techniques (docker, systemd. chroot, qemu, binfmt),
```
Comment (lmolina) [inline README.md:167]:
```suggestion
different containerization techniques (docker, systemd, chroot, qemu, binfmt),
```

File: README.md [hunk 10]
```diff
@@ -1,116 +1,184 @@
...
+check [DevTools](./DevTools/) directory for more.
```
Comment (lmolina) [inline README.md:168]:
```suggestion
check [DevTools](./DevTools/) directory for additional information.
```

File: README.md [hunk 11]
```diff
@@ -1,116 +1,184 @@
...
 
-A helper script allows to setup env and build on device, docker or cloud (eg: github actions).
+Reference setup, is RaspberryPi on raspbian-9 OS (EoL)
 
-```bash
-./helper.mk help
-./helper.mk setup/debian
-./helper.mk
+Dump image to SDcard from:
+
```
Comment (lmolina) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -1,116 +1,184 @@
...
+- http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip
```
Comment (lmolina) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -1,116 +1,184 @@
-# zipgateway
+# Z/IP Gateway SDK
 
-## Licensing
+## LICENSING
 
-ZipGateway is covered by one of several different licenses.
-The default license is the [Master Software License Agreement (MSLA)](https://www.silabs.com/about-us/legal/master-software-license-agreement), which applies unless otherwise noted. 
+Z/IP Gateway is covered by one of several different licenses.
```
[the remark this answers is not in the document]
Comment (rzr) [reply README.md:5]: i am not sure i will have to change it this is SL current policy (but you're right)

see: https://github.com/SiliconLabs/gecko_sdk/blob/gsdk_4.4/License.txt

I will ask clarification to our legal dept

[the remark this answers is not in the document]
Comment (lmolina) [reply README.md:5]: OK, perhaps just rephrase to "The source code of the Z/IP Gateway is covered by one of several different licenses..."

[the remark this answers is not in the document]
Comment (rzr) [reply README.md]: line changed

File: README.md [hunk 14]
```diff
@@ -1,116 +1,164 @@
-# zipgateway
+# Z/IP Gateway
 
-## Licensing
+The Z/IP gateway is a free reference design intended for development and demonstration purposes only. It is provided as is, with no warranties. Customers are advised to conduct security and compliance testing of all gateways during the product design process.
 
-ZipGateway is covered by one of several different licenses.
-The default license is the [Master Software License Agreement (MSLA)](https://www.silabs.com/about-us/legal/master-software-license-agreement), which applies unless otherwise noted. 
-Refer to [LICENSE](./LICENSE) for more details.
+> [!WARNING]
```
Comment (rzr) [inline README.md:5]: ## DISCLAIMER

File: README.md [hunk 15]
```diff
@@ -1,116 +1,164 @@
...
+# Z/IP Gateway
 
-## Licensing
+The Z/IP gateway is a free reference design intended for development and demonstration purposes only. It is provided as is, with no warranties. Customers are advised to conduct security and compliance testing of all gateways during the product design process.
 
-ZipGateway is covered by one of several different licenses.
-The default license is the [Master Software License Agreement (MSLA)](https://www.silabs.com/about-us/legal/master-software-license-agreement), which applies unless otherwise noted. 
-Refer to [LICENSE](./LICENSE) for more details.
+> [!WARNING] 
+> The Z/IP Gateway solution is in maintenance mode. To access a maintained Z-Wave gateway solution supporting the latest protocol features, refer to the [Unify SDK](https://github.com/SiliconLabs/UnifySDK) project.
```
Comment (rzr) [inline README.md:6]:
```suggestion
## DISCLAIMER

The Z/IP Gateway solution is in maintenance mode. To access a maintained Z-Wave gateway solution supporting the latest protocol features, refer to the [Unify SDK](https://github.com/SiliconLabs/UnifySDK) project.
```

**Review by lmolina — APPROVED** [review]

**Pull request merged**